protocol/v2/ssv/validator/startup.go

```diff
@@ -19,6 +18,10 @@ func (v *Validator) Start() error {
 		if !ok {
 			return nil
 		}
+
+		// Initialize the message queue.
+		v.Q = queue.New(nil)
```

Comment:
why nil?

Reply:
There's no point passing a `MessagePrioritizer` at the beginning because we do so each time with `Sort` inside `ConsumeQueue`